examples/examplesVersionDependency.ttl

```diff
@@ -26,10 +36,27 @@
 	] ;
 	.
 
+<https://ontologies.exco.com/unversionedOntology>
+	a owl:Ontology ;
+	skos:scopeNote "This ontology does not declare a version."^^xsd:string ;
+	skos:prefLabel "Unversioned Ontology"^^xsd:string ;
+	.
+
+<https://ontologies.exco.com/unconventionalVersioning>
+	a owl:Ontology ;
+	owl:versionIRI <https://ontologies.exco.com/unconventionalVersioning-RC2> ;
+	skos:scopeNote "This ontology uses a non-conventional version IRI."^^xsd:string ;
```

Comment:
> Like unversionedOntology, this ontology also does not declare a semantic version. Is this worth mentioning? Or is it better for it to have a semantic version number but an unconventional version IRI?
> 
> There is ambiguity when we say 'a version': the versionIRI or the semantic version. Not obvious how best to clean that up. A start would be to always say 'semantic version number' if it is, and maybe refer to an ontology's 'version IRI' or 'IRI version' . So we might say:
> 
> * "This ontology uses an unconventional version IRI" or "This ontology has no semantic version number and has an unconventional IRI version." or
> * "This ontology declares no semantic number and has an unconventional IRI version."

This can be ignored, too fiddly to worry about.